frontend/amundsen_application/static/js/features/ColumnList/index.tsx

```diff
@@ -17,7 +17,7 @@ import {
   notificationsEnabled,
   getMaxLength,
   getTableSortCriterias,
-  isColumnListLineageEnabled,
+  isColumnListLineageEnabled, getMaxNestedColumns,
```

Comment:
I have set my VSCode to do formatting on save, you probably would benefit from it!